src/main/java/org/opensearch/neuralsearch/processor/util/RestActionUtils.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.neuralsearch.processor.util;
+
+import org.opensearch.rest.RestRequest;
+
+import java.util.Optional;
+
+public class RestActionUtils {
```

Comment:
```suggestion
class RestActionUtils {
```

Make this class package-private